src/ripple/json/impl/Writer.cpp

```diff
@@ -244,20 +243,20 @@ void Writer::output (std::string const& s)
 void Writer::output (Json::Value const& value)
 {
     impl_->markStarted();
-    writeJson (value, impl_->getOutput());
+    outputJson (value, impl_->getOutput());
 }
 
 template <>
 void Writer::output (float f)
 {
```

Comment:
These template specializations need to be inlined or Visual Studio will complain about multiple definitions at link time. Change to `inline void ...`